format/spec.md

```diff
@@ -965,10 +982,31 @@ Types are serialized according to this table:
+| **`float`**        | **`json float`**   | `1.1`                                    |                                                                                                                                                                                                                                                                                                                           |
+| **`double`**       | **`json double`**  | `1.1`                                    |                                                                                                                                                                                                                                                                                                                           |
+| **`decimal(P,S)`** | **`jaso number`**  | `123.45`                                 | Stores the decimal value as a general json number                                                                                                                                                                                                                                                                         |
+| **`date`**         | **`string`**       | `"2007-12-03"`                           | Stores ISO-8601 standard date                                                                                                                                                                                                                                                                                             |
+| **`time`**         | **`string`**       | `"10:15:30"`                             | Stores ISO-8601 standard time                                                                                                                                                                                                                                                                                             |
+| **`timestamp`**    | **`string`**       | `"2007-12-03T10:15:30"`                  | Stores ISO-8601 standard date-time without time-zone                                                                                                                                                                                                                                                                      |
+| **`timestamptz`**  | **`string`**       | `"2007-12-03T10:15:30"`                  | Stores ISO-8601 standard date-time with time-zone                                                                                                                                                                                                                                                                         |
```

Comment:
This is an invalid `timestamptz` value. Timestamp with time zone literals are required to have a zone offset.

In addition, the examples should always include precision down to microseconds for `time`, `timestamp`, and `timestamptz`. Even if the parser is more lax and allows values with different precision, the spec should always represent values correctly.